app/assets/javascripts/feedback.js

```diff
@@ -0,0 +1,14 @@
+document.getElementById("pageBody").style.width = '90%';
+$(window).scroll(function(e){ 
+  const $el = $('.result-fixed'); 
+  const container = $('.result-container')[0];
+  const containerWidth = container.offsetWidth;
+  if (container.getBoundingClientRect().top <= 30){ 
+    // set to be 25% of feedback container width instead of entire page
+    // (which is what happens if you use % for a fixed div)
+    // if the .25 is changed, also change the width for .result-summary class
+    $el.css({'position': 'fixed', 'top': '30px', width: `${containerWidth * .25}px`}); 
```

Comment:
May I know what is this fixed positioning for? I'm assuming it's the results box

Reply:
Yes, it's to keep it floating at the top; Ideally, the width would be set in the CSS, but there's some weirdness going on with how % is calculated with fixed positioning and it causes it to bug out that way